src/middlewared/middlewared/alert/source/dif_formatted_disks.py

```diff
@@ -0,0 +1,14 @@
+from middlewared.alert.base import Alert, AlertClass, AlertCategory, OneShotAlertClass, AlertLevel
+
+
+class DifFormattedAlertClass(AlertClass, OneShotAlertClass):
+    category = AlertCategory.HARDWARE
+    level = AlertLevel.CRITICAL
+    title = 'Disk(s) Are Formatted With Data Integrity Feature (DIF).'
+    text = 'Disk(s): %s are formatted with Data Integrity Feature (DIF) which is unsupported.'
+
+    async def create(self, disks):
+        return Alert(DifFormattedAlertClass, ', '.join(disks))
```

Comment:
Can you please add `key=None`? Otherwise it will create a new alert each time an erroneous disks set is changed.

Reply:
done